app/components/agenda/agenda-header.js

```diff
@@ -21,324 +12,38 @@ export default class AgendaHeader extends Component {
-    this.args.meeting.releasedDecisions = new Date();
-    this.args.meeting.save();
-  }
-
-  @action
-  openConfirmReleaseDocuments() {
-    this.showConfirmReleaseDocuments = true;
-  }
-
-  @action
-  cancelReleaseDocuments() {
-    this.showConfirmReleaseDocuments = false;
-  }
-
-  @action
-  releaseDocuments() {
-    this.showConfirmReleaseDocuments = false;
-    this.args.meeting.releasedDocuments = new Date();
-    this.args.meeting.save();
-  }
-
-  @action
-  openConfirmPublishThemis() {
-    this.showConfirmPublishThemis = true;
-  }
-
-  @action
-  cancelPublishThemis() {
-    this.showConfirmPublishThemis = false;
-  }
-
-  @task
-  *publishThemis(scope) {
-    try {
-      const themisPublicationActivity = this.store.createRecord('themis-publication-activity', {
-        startDate: new Date(),
-        meeting: this.args.meeting,
-        scope
-      });
-      yield themisPublicationActivity.save();
-      yield this.loadLatestPublicationActivity.perform();
-      this.toaster.success(this.intl.t('success-publish-to-web'));
-    } catch(e) {
-      this.toaster.error(
-        this.intl.t('error-publish-to-web'),
-        this.intl.t('warning-title')
-      );
-    }
-    this.showConfirmPublishThemis = false;
-  }
-
-  @action
-  openConfirmUnpublishThemis() {
-    this.showConfirmUnpublishThemis = true;
-  }
-
-  @action
-  cancelUnpublishThemis() {
-    this.showConfirmUnpublishThemis = false;
-  }
-
-  @task
-  *unpublishThemis(scope) {
-    try {
-      const themisPublicationActivity = this.store.createRecord('themis-publication-activity', {
-        startDate: new Date(),
-        meeting: this.args.meeting,
-        scope
-      });
-      yield themisPublicationActivity.save();
-      this.toaster.success(this.intl.t('success-unpublish-from-web'));
-    } catch(e) {
-      this.toaster.error(
-        this.intl.t('error-unpublish-from-web'),
-        this.intl.t('warning-title')
-      );
-    }
-    this.showConfirmUnpublishThemis = false;
-  }
-
-  @action
-  toggleEditingMeeting() {
-    this.isEditingMeeting = !this.isEditingMeeting;
-  }
-
-  @action
-  print() {
-    window.print();
-  }
-
-  @action
-  openConfirmApproveAllAgendaitems() {
-    this.reloadAgendaitemsData.perform();
-    this.showConfimApprovingAllAgendaitems = true;
-  }
-
-  @action
-  cancelApproveAllAgendaitems() {
-    this.showConfimApprovingAllAgendaitems = false;
-  }
-
-  @action
-  async approveAllAgendaitems() {
-    this.showConfimApprovingAllAgendaitems = false;
-    this.toggleLoadingMessage(
-      this.intl.t('approve-all-agendaitems-message')
-    );
-    const allAgendaitemsNotOk = await this.args.currentAgenda.get(
-      'allAgendaitemsNotOk'
-    );
-    for (const agendaitem of allAgendaitemsNotOk) {
-      try {
-        await setAgendaitemFormallyOk(agendaitem);
-      } catch {
-        await agendaitem.rollbackAttributes();
-      }
+    if (typeof this.args.onStartLoading === 'function') {
```

Comment:
Not really, I just ran into it throwing exceptions while working on the ticket (because I forgot to pass in the arguments), and I'm used to explicitly checking if something is valid/can be used/... before calling it/passing it as an argument/..., so I default to writing those kinds of checks. Will remove though,